app/models/match_decisions/schedule_criminal_hearing_housing_subsidy_admin.rb

```diff
@@ -99,9 +99,9 @@ def no_hearing
         @decision.next_step.next_step.initialize_decision!
       end
 
-      def canceled
+      def canceled contact
         Notifications::MatchCanceled.create_for_match! match
-        match.canceled!
+        match.canceled! contact&.id
```

Comment:
`ClientOpportunityMatch#canceled!` takes a named parameter, so this won't work as written.

Reply:
changed.